pages/1.11/release-notes/1.11.1/index.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for 1.11.1
+title: Release Notes for 1.11.1
+menuWeight: 0
+excerpt:
+---
+
+These are the release notes for DC/OS 1.11.1.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.1/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+# <a name="issues-fixed"></a>Improvements and Issues Fixed in DC/OS 1.11.1 
+
+- INFINITY-3331 - Fixed cleaning up other framework's volumes.
```

Comment:
I tried to understand this and talked to @gabrielhartmann -- this should be removed because it's covered in SDK release  notes.